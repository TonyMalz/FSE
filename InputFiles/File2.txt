2,1,1,2
3,1,7,8,9
1, ,U
