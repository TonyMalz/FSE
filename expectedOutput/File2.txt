1 : U -> {1, 2, 7, 8, 9}
|
+- 2 : {1, 2} -> {1, 2}
|
+- 3 : {7, 8, 9} -> {7, 8, 9}